src/VscPreviewExtension/src/previewCore.ts

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+import { workspace, window, ExtensionContext, Uri } from "vscode";
+import * as childProcess from "child_process";
+import { MarkdownDocumentContentProvider } from "./markdownDocumentContentProvider";
+
+// Create a child process(DfmRender) by "_spawn" to render a html
+export class PreviewCore {
+    public _isFirstTime: boolean;
+    public _provider: MarkdownDocumentContentProvider;
```

Comment:
Agree!

---
In reply to: [75796570](https://github.com/dotnet/docfx/pull/616#discussion_r75796570) [](ancestors = 75796570)